### PR TITLE
First Commit

### PR DESCRIPTION
**Pull request by cs-shubham-morankar**
### Descriptions:

- Added the following actions:
   - Create Application
   - Get Applications
   - Get Application by Name
   - Update Application
   - Delete Application
   - Get Clusters

#### QA Impact:
- [x] `full`
- [x] Ensure that platform dependencies remain unaffected

#### UTCs:
- [x] Connector installation verified
- [x] Connector logo verified
- [x] Check health verified
- [x] Actions and Playbooks list verified
- [x] Playbook tags verified
- [x] All playbooks are in info mode verified
- [x] All playbooks are in inactive mode verified
- [x] Verified platform dependencies remain unaffected

#### Pytest Test Cases: (If applicable)
- [ ] Attach the test report

### CODE REVIEW

File: argo-cd/info.json [hunk 1]
```diff
@@ -0,0 +1,894 @@
...
+  "label": "Argo CD",
+  "description": "Argo CD (short for Argo Continuous Delivery) is a declarative, GitOps continuous delivery tool for Kubernetes. It allows you to manage and deploy applications to Kubernetes clusters using Git repositories as the source of truth for both the desired application state and its configuration.",
+  "publisher": "Fortinet",
+  "cs_approved": true,
+  "cs_compatible": true,
+  "contributor": "",
+  "help_online": "",
+  "icon_small_name": "small.png",
+  "icon_large_name": "large.png",
+  "category": "DevOps and Digital Operations",
```
Comment (cs-harsh-sharma) [inline argo-cd/info.json:13]: Is this category allowed or are we planning to add this category ??

File: argo-cd/info.json [hunk 2]
```diff
@@ -0,0 +1,894 @@
...
+        "value": true,
+        "tooltip": "Specifies whether the SSL certificate for the server is to be verified. By default, this option is set to True."
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "create_application",
+      "title": "Create Application",
+      "description": "Creates a application in Argo CD based on the application name, source, destination, and other input parameters that you have specified.",
```
Comment (cs-harsh-sharma) [inline argo-cd/info.json]: minor change in description..
"Creates a application" -> "Creates an application"

File: argo-cd/info.json [hunk 3]
```diff
@@ -0,0 +1,894 @@
...
+          "description": "Specify the operation, in the JSON format, based on which you want to create application in Argo CD.",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "json",
+          "name": "operation",
+          "tooltip": "Specify the operation, in the JSON format, based on which you want to create application in Argo CD."
+        },
+        {
+          "title": "Additional Properties",
```
Comment (cs-harsh-sharma) [inline argo-cd/info.json:140]: ***Suggestion:

It would be good to have example/sample JSON for other params too...

File: argo-cd/info.json [hunk 4]
```diff
@@ -0,0 +1,894 @@
...
+        },
+        {
+          "title": "Destination",
+          "description": "Specify the destination, in the JSON format, based on which you want to update application in Argo CD.",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "json",
+          "name": "destination",
+          "tooltip": "Specify the destination, in the JSON format, based on which you want to retrieve application in Argo CD.",
```
Comment (cs-harsh-sharma) [inline argo-cd/info.json]: Please update the tooltip

File: argo-cd/playbooks/playbooks.json [hunk 5]
```diff
@@ -0,0 +1,550 @@
...
+          "versions": [],
+          "triggerStep": "/api/3/workflow_steps/1f6858bd-d2f0-44fe-9558-3fc572c5d2bb",
+          "steps": [
+            {
+              "@type": "WorkflowStep",
+              "name": "Get Application by Name",
+              "description": null,
+              "arguments": {
+                "name": "Argo CD",
+                "config": "''",
```
Comment (cs-harsh-sharma) [inline argo-cd/playbooks/playbooks.json]: Please remove quotes

File: argo-cd/info.json (hunk 1 above)
Comment (cs-shubham-morankar) [reply argo-cd/info.json:13]: As discuss, provided category is correct.

**Review by cs-harsh-sharma — APPROVED** [review]

**Pull request merged**